src/dev/code_coverage/ingest_coverage/__tests__/transforms.test.js

```diff
@@ -31,4 +31,32 @@ describe(`Transform fn`, () => {
       expect(ciRunUrl({ a: 'a' })).not.to.have.property('ciRunUrl');
     });
   });
+  describe(`coveredFilePath`, () => {
+    it(`should remove the jenkins workspace path`, () => {
+      const obj = {
+        staticSiteUrl:
+          '/var/lib/jenkins/workspace/elastic+kibana+code-coverage/kibana/x-pack/legacy/plugins/reporting/server/browsers/extract/unzip.js',
+        COVERAGE_INGESTION_KIBANA_ROOT:
```

Comment:
Is this the place you would test a path which contains 'kibana' ?  We had the problem with the greedy regexp which was cutting out too much.

Reply:
yes sir, it was fixed before the original pr was merged into master.  But, that is indeed why I added this test.

Reply:
sorry, I don't understand.  I'm thinking the test should have 'kibana' in the path like;

https://kibana-coverage.elastic.dev/2020-05-19T13:33:03Z/jest-combined/src/plugins/kibana_usage_collection/server/collectors/kibana/get_saved_object_counts.ts.html

to test that it gets parsed to;

src/plugins/kibana_usage_collection/server/collectors/kibana/get_saved_object_counts.ts

and not to (up to embedded 'kibana');

get_saved_object_counts.ts

Reply:
I don't have the right staticSiteURL there, but the idea is that there was a problem with the regular expression that was eating too much of the file path up to 'kibana'